staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -87,8 +87,9 @@ var (
 	//
 	// Can we make these passable to NewGenericWebhook?
 	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
-	groupVersions        = []schema.GroupVersion{auditv1alpha1.SchemeGroupVersion}
-	registry             = registered.NewOrDie("")
+	// TODO(audit): figure out a general way to let the client choose their preferred version
+	groupVersions = []schema.GroupVersion{auditv1beta1.SchemeGroupVersion}
```

Comment:
Right. @CaoShuFeng maybe misunderstanding. With removing the change I only meant those few lines around `func init()` to make sure that at least one GroupVersion is registered. The other parts of my commits should go in.